uktrade/data-hub-frontend
# Replace Body component with GOVUK React Main

## Description of change

This PR replaces the Body component with the GOVUK React Main component, in the files which used the recently added react local header. Originally, the Body component was developed in order to maintain styling that was previously provided by a Nunjucks template. It turns out that the GOVUK React Main component does that for us. This PR simply swaps those out. 

## Test instructions

If you navigate to the Send Referrals page, you should see no change in the styling of the local header, compared to the other headers which do not use React.

## Checklist

[//]: # "When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/master/docs/Code%20review%20guidelines.md"
